data/json/items/classes/comestible.json

```diff
@@ -0,0 +1,61 @@
+[
+  {
+    "id": "food",
+    "type": "COMESTIBLE",
+    "name": "edible comestible",
+    "comestible_type": "FOOD",
+    "volume": 1,
+    "symbol": "%"
+  },
+  {
+    "id": "fruit",
+    "copy-from": "food",
+    "type": "COMESTIBLE",
+    "name": "base fruit",
+    "material": "fruit",
+    "color": "red",
+    "vitamins": [ [ "vitC", 15 ] ]
+  },
+  {
+    "id": "fruit_berry",
+    "copy-from": "fruit",
+    "type": "COMESTIBLE",
+    "name": "fruit berries",
+    "stack_size": 8,
```

Comment:
IRL berries are quite small. Stack size 4-8 would represent their size better.

Alternatively, name change from "raspberry" to "handful of raspberries" to avoid having to change numbers in recipes.